run_benchmarks.py

```diff
@@ -0,0 +1,444 @@
+#!/usr/bin/env python3
+
+import argparse
+import io
+import logging
+import os
+import re
+import shutil
+import signal
+import sys
+import time
+import unittest
+from glob import glob
+from multiprocessing import Process
+from os.path import abspath, relpath
+from subprocess import PIPE, STDOUT, Popen
+from unittest import TestLoader, TextTestResult, TextTestRunner
+from unittest.mock import patch
+
+import pandas as pd
+
+logging.basicConfig(
+    level=logging.INFO,
+    format="%(asctime)s - %(levelname)s - %(message)s",
+)
+
+
+class BenchRunner:
+    def __init__(self, base_dir="build/Release"):
```

Comment:
Better to use `pathlib` everywhere when working with paths in python, instead of `str`